src/plugins/imagecapturer/CImageCapturer.cpp

```diff
@@ -114,6 +119,10 @@ namespace semrec {
       try {
 	if(sensor_msgs::image_encodings::isColor(m_imgReceived.encoding)) {
 	  cv_ptr = cv_bridge::toCvCopy(m_imgReceived, sensor_msgs::image_encodings::BGR8);
+  } else if(isDepthImage(m_imgReceived.encoding)) {
```

Comment:
I guess such that it looks the same on GitHub. I try to keep the indentation consistent, but if your settings don't match the ones reflected on the repo, I'll fix them later.

I'll just merge it in.